cmd/cmd_renew.go

```diff
@@ -240,6 +313,45 @@ func needRenewal(x509Cert *x509.Certificate, domain string, days int) bool {
 	return true
 }
 
+// needRenewalARI checks if the certificate needs to be renewed using the
+// renewalInfo endpoint.
+func needRenewalARI(ctx *cli.Context, cert, issuer *x509.Certificate, domain string, client *lego.Client) *time.Time {
```

Comment:
nit: this function name was great when it returned a boolean, is slightly confusing now that it returns a timestamp